build/testing/cli.go

```diff
@@ -273,6 +273,108 @@ exit $?`,
 		}
 	}
 
+	{
+		container = container.Pipeline("flipt bundle").
+			WithWorkdir("build/testing/testdata/bundle")
+
+		var err error
+		container, err = assertExec(ctx, container, flipt("bundle", "build", "mybundle:latest"),
+			stdout(matches(`sha256:[a-f0-9]{64}`)))
+		if err != nil {
+			return err
+		}
+
+		container, err = assertExec(ctx, container, flipt("bundle", "list"),
+			stdout(matches(`DIGEST[\s]+REPO[\s]+TAG[\s]+CREATED`)),
+			stdout(matches(`[a-f0-9]{7}[\s]+mybundle[\s]+latest[\s]+[0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2}`)))
+		if err != nil {
+			return err
+		}
+
+		// we need to wait for a second to ensure the timestamp ticks over so that
+		// a new digest is created on the second build
+		time.Sleep(time.Second)
```

Comment:
I think it will be determinstic enough. We just need the clock to tick one second to create a new digest from the manifests. I think it should halt the process for at-least one wall clock second. So it should always result in a different SHA.